packages/rs-drive-abci/tests/strategy_tests/main.rs

```diff
@@ -780,6 +780,8 @@ mod tests {
 
     #[test]
     fn run_chain_insert_one_new_identity_per_block() {
```

Comment:
Change the title to run_chain_insert_one_new_identity_per_block_with_block_signing